home/lncm/compose/tor/docker-compose.yml

```diff
@@ -1,9 +1,11 @@
 version: '3.7'
 services:
   bitcoind:
-    image: lncm/bitcoind:0.17.1
+    image: lncm/bitcoind:0.18.0-linux-arm
+    entrypoint: bitcoind -zmqpubrawblock=tcp://0.0.0.0:28332 -zmqpubrawtx=tcp://0.0.0.0:28333
```

Comment:
We already have `CMD` set to the same.  Specifying ZMQ endpoints here shouldn't be necessary. 

See: https://github.com/lncm/docker-bitcoind/blob/master/0.17/Dockerfile#L92

Reply:
It didn't work when I building it so I added it in for good measure